src/empty.js

```diff
@@ -26,9 +27,10 @@ module.exports = _curry1(function empty(x) {
     return x.constructor.empty();
   } else {
     switch (Object.prototype.toString.call(x)) {
-      case '[object Array]':  return [];
-      case '[object Object]': return {};
-      case '[object String]': return '';
+      case '[object Arguments]':  return (function() { return arguments; }());
```

Comment:
Heads up, this isn't backwards compatible to IE7 where `toString(arguments) == '[object String]'` or something